archtitus.sh

```diff
@@ -17,28 +27,27 @@ echo -ne "
 -------------------------------------------------------------------------
                     Automated Arch Linux Installer
 -------------------------------------------------------------------------
-                Scripts are in directory named ArchTitus
 "
-    ( bash $SCRIPT_DIR/scripts/startup.sh )|& tee startup.log
-      source $CONFIGS_DIR/setup.conf
-    ( bash $SCRIPT_DIR/scripts/0-preinstall.sh )|& tee 0-preinstall.log
-    ( arch-chroot /mnt $HOME/ArchTitus/scripts/1-setup.sh )|& tee 1-setup.log
+}
+
+sequence() {
+    . $SCRIPT_DIR/scripts/startup.sh # Dont need to log user password in plain text
```

Comment:
The log file doesn't contain the password, though. It's just what's output in the terminal from the script. Your input is masked in terminal.